### PR TITLE
test: update nightly test paths

### PR DESCRIPTION
**Pull request by aaron276h**

### CODE REVIEW

File: e2e_tests/tests/nightly/test_convergence.py [hunk 1]
```diff
@@ -56,9 +56,9 @@ def test_mnist_estimator_accuracy() -> None:
 
 @pytest.mark.nightly  # type: ignore
 def test_mnist_pytorch_accuracy() -> None:
-    config = conf.load_config(conf.cv_examples_path("mnist_pytorch/const.yaml"))
+    config = conf.load_config(conf.cv_examples_path("mnist_multi_output_pytorch/const.yaml"))
     experiment_id = exp.run_basic_test_with_temp_config(
-        config, conf.cv_examples_path("mnist_pytorch"), 1
+        config, conf.cv_examples_path("mnist_multi_output_pytorch"), 1
```
Comment (mackrorysd) [inline e2e_tests/tests/nightly/test_convergence.py:61]: blocking: Looking at PR #1301, I'm pretty sure this was supposed to be tutorials_path("mnist_pytorch"), not cv_examples_path("mnist_multi_output_pytorch").

**Review by mackrorysd — CHANGES_REQUESTED** [review]

File: e2e_tests/tests/nightly/test_convergence.py (hunk 1 above)
Comment (naren-determined) [reply e2e_tests/tests/nightly/test_convergence.py:61]: Yes ^

**Review by naren-determined — COMMENTED** [review]
One more favor: can you update the top-level examples README to have `# Determined Examples` as a title? Forgot to do that sorry.

File: e2e_tests/tests/nightly/test_convergence.py (hunk 1 above)
Comment (aaron276h) [reply e2e_tests/tests/nightly/test_convergence.py]: fixed

**Comment by aaron276h** [comment]
@naren-determined i updated the README

**Review by mackrorysd — APPROVED** [review]

**Review by naren-determined — APPROVED** [review]
Sweet!

**Pull request merged**